server/model/index.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/alerts_db', {useNewUrlParser: true});
+
+const alertsDb = mongoose.connection;
+
+alertsDb.on('open', () => {
```

Comment:
You could have a skipped the assignment of the connection to the variable alertsDb, since you want to set a listener and then you're not doing anything else with alertsDb:
```javascript
mongoose.connection.on('open', () => {...
```